cmd/collection/main.go

```diff
@@ -90,62 +90,68 @@ func main() {
+			"expiry buffer for inbound transactions")
+		flags.UintVar(&ingestConf.PropagationRedundancy, "ingest-tx-propagation-redundancy", 10,
+			"how many additional cluster members we propagate transactions to")
+		flags.Uint64Var(&ingestConf.MaxAddressIndex, "ingest-max-address-index", 10_000_000,
+			"the maximum address index allowed in transactions")
+		flags.UintVar(&builderExpiryBuffer, "builder-expiry-buffer", builder.DefaultExpiryBuffer,
+			"expiry buffer for transactions in proposed collections")
+		flags.Float64Var(&builderPayerRateLimit, "builder-rate-limit", builder.DefaultMaxPayerTransactionRate, // no rate limiting
+			"rate limit for each payer (transactions/collection)")
+		flags.StringSliceVar(&builderUnlimitedPayers, "builder-unlimited-payers", []string{}, // no unlimited payers
+			"set of payer addresses which are omitted from rate limiting")
+		flags.UintVar(&maxCollectionSize, "builder-max-collection-size", flow.DefaultMaxCollectionSize,
+			"maximum number of transactions in proposed collections")
+		flags.Uint64Var(&maxCollectionByteSize, "builder-max-collection-byte-size", flow.DefaultMaxCollectionByteSize,
+			"maximum byte size of the proposed collection")
+		flags.Uint64Var(&maxCollectionTotalGas, "builder-max-collection-total-gas", flow.DefaultMaxCollectionTotalGas,
+			"maximum total amount of maxgas of transactions in proposed collections")
+		flags.DurationVar(&hotstuffTimeout, "hotstuff-timeout", 60*time.Second,
+			"the initial timeout for the hotstuff pacemaker")
+		flags.DurationVar(&hotstuffMinTimeout, "hotstuff-min-timeout", 2500*time.Millisecond,
+			"the lower timeout bound for the hotstuff pacemaker")
+		flags.Float64Var(&hotstuffTimeoutIncreaseFactor, "hotstuff-timeout-increase-factor",
+			timeout.DefaultConfig.TimeoutIncrease,
+			"multiplicative increase of timeout value in case of time out event")
+		flags.Float64Var(&hotstuffTimeoutDecreaseFactor, "hotstuff-timeout-decrease-factor",
+			timeout.DefaultConfig.TimeoutDecrease,
+			"multiplicative decrease of timeout value in case of progress")
+		flags.Float64Var(&hotstuffTimeoutVoteAggregationFraction, "hotstuff-timeout-vote-aggregation-fraction",
+			timeout.DefaultConfig.VoteAggregationTimeoutFraction,
+			"additional fraction of replica timeout that the primary will wait for votes")
+		flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
+			"the delay to broadcast block proposal in order to control block production rate")
+
+		// epoch qc contract flags
+		flags.StringVar(&accessAddress, "access-address", "", "the address of an access node")
+		flags.StringVar(&secureAccessNodeID, "secure-access-node-id", "", "the node ID of the secure access GRPC server")
+		flags.BoolVar(&insecureAccessAPI, "insecure-access-api", true, "required if insecure GRPC connection should be used")
+	})
+
+	err = nodeBuilder.Initialize()
+	if err != nil {
+		nodeBuilder.Logger.Fatal().Err(err).Send()
+	}
```

Comment:
nit:

```suggestion
	if err = nodeBuilder.Initialize(); err != nil {
		nodeBuilder.Logger.Fatal().Err(err).Send()
	}
```

Simlilar suggestion for all the other places